_posts/2022-10-26-basel-one-conference-report.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 'BaselOne Conference Report'
+author: markus
+featuredImage: BaselOne22
+excerpt: "BaselOne 22 took place on Oct 19, 2022 in the Markthalle in Basel, Switzerland. Let us have a look back at this amazing event from different views: as one of the organizers, as speaker and as attendees."
+permalink: '/2022/10/26/baselone22.html'
+categories: [Java, Metrics, AI, NLP, Conferences, Kubernetes, Docker, DevOps, Istio, Service Mesh]
+header:
+  text: BaselOne22 Conference Report
+  image: post
+---
+
+BaselOne 2022 conference took place on October 19, 2022 in the Markthalle. As we were quite a few people from Karakun attending, be it as participants, speakers or organizer, we would like to share our impressions in this blog post. Some hard facts first:
+
+## Some figures about BaselOne22 
+BaselOne is a yearly event organized by the eponymous association existing of [Karakun AG](https://karakun.com), [Bluesky IT-Solutions AG](https://bluesky-it.ch) and [Baloise Insurance](https://baloise.com).
+
+In total, BaselOne 22 offered 26 talks by 29 speakers and 4 workshops to 160 paying attendees. All in all, almost 400 hours of organization time were spent to make it happen.
+
+
+
+## Day 1 - Conference Day
+
+As said, there were 26 talks, and it was challenging to choose which one to attend. The lineup was packed with well-known speakers like Ixchel Ruiz, Falk Sippach, Sandra Parsick, Otavio Santana, Katrin Rabow, Gerd Aschemann, Chris Engelbert and many more. The talks were nicely distributed across variety of topics and domains of IT. It is worth to mention that most of the talks had an introductory flavour to it to attract more beginners, but also to light a spark among more experienced folks. From more advanced talks to one of our colleague [George](https://twitter.com/grigala_) clearly stand out a talk about Service Mesh by [Abdellfetah Sghiouar](https://twitter.com/boredabdel):
```

Comment:
I suggest linebreaks and think the last sentence had a typo:
```suggestion
As said, there were 26 talks, and it was challenging to choose which one to attend. 
The lineup was packed with well-known speakers like Ixchel Ruiz, Falk Sippach, Sandra Parsick, Otavio Santana, Katrin Rabow, Gerd Aschemann, Chris Engelbert and many more.
The talks were nicely distributed across variety of topics and domains of IT. 
It is worth to mention that most of the talks had an introductory flavour to it to attract more beginners, but also to light a spark among more experienced folks. 
From the more advanced talks to one of our colleague [George](https://twitter.com/grigala_) clearly stands out a talk about Service Mesh by [Abdellfetah Sghiouar](https://twitter.com/boredabdel):
```

Reply:
lgtm

Reply:
lgtm